easybuild/easyblocks/p/python.py

```diff
@@ -279,6 +279,21 @@ def install_step(self):
         if self.cfg['ebpythonprefixes']:
             write_file(os.path.join(self.installdir, self.pythonpath, 'sitecustomize.py'), SITECUSTOMIZE)
 
+        # symlink lib/python*/lib-dynload to lib64/python*/lib-dynload if it doesn't exist;
+        # see https://github.com/easybuilders/easybuild-easyblocks/issues/1957
+        lib_dynload = 'lib-dynload'
+        lib_dynload_path = os.path.join(self.installdir, 'lib', 'python%s' % self.pyshortver, lib_dynload)
+        if not os.path.exists(lib_dynload_path):
+            lib64_dynload_path = os.path.join(self.installdir, 'lib64', 'python%s' % self.pyshortver, lib_dynload)
+            if os.path.exists(lib64_dynload_path):
+                lib_dynload_parent = os.path.dirname(lib_dynload_path)
+                mkdir(lib_dynload_parent, parents=True)
+                cwd = change_dir(lib_dynload_parent)
+                # use relative path as target, to avoid hardcoding path to install directory
+                target_lib_dynload = os.path.join('..', '..', 'lib64', 'python%s' % self.pyshortver, lib_dynload)
```

Comment:
In line with your usual comments for code that repeats itself...
put os.path.join('lib64', 'python%s' % self.pyshortver, lib_dynload))
into some variable and reuse in both places
or even the ('python%s' % self.pyshortver, lib_dynload) part...

Reply:
fixed in b73b6ed47